taqueria-plugin-contract-types/tasks.ts

```diff
@@ -30,9 +30,17 @@ const generateContractTypes = (parsedArgs: Opts) =>
 		return `${contractFilename}: Types generated`;
 	};
 
+const isContractFile = (filename: string): boolean => {
+	return !filename.includes('.default_storage.')
+		&& !filename.includes('.storage.')
```

Comment:
It's nitpicking, but if someone names their contract like `build.storage.contract.jsligo`, this will exclude them.
But our convention for storage is that it should be right before the extension. Am I right?

Reply:
oh it will and should exclude `build.storage.contract.jsligo`

Reply:
it's CONTRACT.storage.NAME.jsligo

Reply:
Ah OK. Bad example. Yes. Now I remember.